packages/graphql/tests/schema/comments.test.ts

```diff
@@ -1597,6 +1611,7 @@ describe("Comments", () => {
                   moviesConnection(after: String, first: Int, sort: [MovieSort], where: MovieWhere): MoviesConnection!
                 }
 
+                \\"\\"\\"\\"\\"\\"
```

Comment:
`QueryOptions` appears to have an empty description!

Reply:
Can you give a suggestion about what it should be? :)